Index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
```

Comment:
I, Thaofeeqat Dauda worked on this project  with Pierre creating the index html and css file.